### PR TITLE
new branch created to get rid of merge conflicts and add more functio…

### PR DESCRIPTION
**Pull request by katiyelae**
…nality

# Pull Request Template

## Description

The last branch pushed had some merge conflicts, so this one is to replace that one. 

## Checklist

- [ ] I have tested these changes locally to ensure they work as expected.
- [ ] I have reviewed and addressed any code comments, suggestions, or feedback.
- [ ] I have updated the project documentation if these changes affect it.
- [ ] I have resolved any merge conflicts (if applicable).
- [ ] I have run automated tests, and they pass without errors.
- [ ] I have squashed my commits into a single, coherent commit (if necessary).

## Related Issues

Closes #[Issue Number]

If your pull request resolves a specific issue, reference it here using the `Closes #` syntax.

## Screenshots (if applicable)

Include screenshots or GIFs that demonstrate the changes visually, especially for UI-related changes.

## Additional Notes

Provide any additional information, context, or details that you think the reviewers or the project maintainers should be aware of. This could include any known limitations or potential future work related to this PR.

## Reviewers

- @funsav3r
- @IrascibleMidnight
- @juhi1103
- @ilgincevik
- @tihardy
- @erinaperez
- @contributor7
- @contributor8

Assign the PR to relevant reviewers from the contributors list. Reviewers, please ensure that the changes meet project standards, follow coding guidelines, and fulfill the project's requirements.

### CODE REVIEW

File: src/components/buttons/GetRecsButton.jsx [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
+  };
+  const handleRecButton = () => {
+    onClickRecButton(); //// NOTE this should go before navigate so that it passes the data first before navigating
+    /// NOTE since this fn just does the one thing, why not just call it in the handler rather than abstracting it into another function?
+    // navigateToRecommendations();
+    navigate('/Recs');
+
+    // NOTE idk what was the intention of the below is, looks like a mistake? This was what broke it
+    // {
+    //   onClickRecButton;
```
Comment (ann-codes) [inline src/components/buttons/GetRecsButton.jsx]: This was what broke it

File: src/routes/Recs.jsx [hunk 2]
```diff
@@ -17,38 +19,63 @@ export default function Recommendations() {
   const thumbsUp = () => {
     setShowButton(false);
   };
+  // console.log(finalSummary);
+
+  //// NOTE sample below
```
Comment (ann-codes) [inline src/routes/Recs.jsx]: @katiyelae suggestion below on filtering

File: src/static/recommendationData.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
-export const recommendationData = [
+const RecsData = [
   {
     type: 'Exercise',
     activity: 'Walking (alone)',
-    needsEquipment: false,
+    needsEquipment: 'No',
```
Comment (ann-codes) [inline src/static/recommendationData.js:5]: Curious why was this changed to strings? You can check booleans in a similar way:
```js
if (rec.needsEquipment === true) {
// do thing
}

// or short circuiting looks like 
if (rec.needsEquipment) {
// do thing when true
}

if (!rec.needsEquipment) {
// do thing when false
}
```

File: src/routes/Recs.jsx [hunk 4]
```diff
@@ -17,38 +19,40 @@ export default function Recommendations() {
   const thumbsUp = () => {
     setShowButton(false);
   };
+  // console.log(finalSummary);
+
+  //// What does the code do? It gets the description based on finalScore, lowercase it, then splits it into arr items by space, and keeps only the first word
 
-  console.log(showButton);
+  const energyRequiredBasedOnFinalPoints = sumOfPoints(finalScore).toLocaleLowerCase().split(' ')[0];
+  console.log(energyRequiredBasedOnFinalPoints);
```
Comment (ann-codes) [inline src/routes/Recs.jsx:27]: should probably clean up these logs

**Review by ann-codes — APPROVED** [review]

**Review by tihardy — APPROVED** [review]
